business_notes.md

```diff
@@ -28,3 +29,9 @@
         * NOT FUNDED - undetermined loan not funded. 
         * Undisbursed comes before cancelled
 
+
+
+
+Questions:
+1. What is the SBA data refresh cycle like? Does all of the source data need to be up-to-date? Once a year? 
```

Comment:
@SuperNerb I think mentioned that SBA releases updates to the FOIA loan data sets once a quarter?

Reply:
Added this note. Thanks!